### ISSUE TITLE
MeMoModel class is needed

### ISSUE THREAD

**Issue by Porthmeus** [Porthmeus/MeMoMe#42]
While writing code for the metabolite parsing from SBML files, I realized that it would be also handy to have a model class, which does certain tasks on the whole list of metabolites and not only on a single metabolite - this safes a lot of requests if we use online resources to annotate anything.
Here is the general Idea - updates come as I think about it:
```
Class: MeMoModel
Attributes: 
    - metabolites: [MeMoMetabolites]
    - model: [cobra.Model]
Methods:
    - __init__: generic constructor
    - fromPath: __init__(SBMLfile:Path)
    - fromModel:__init__(Model:cobra.Model)
    - fromSBML:__init__(Model:libsbml.Model)
    - annotateAll(self): run all annotation methods
    - different annotation methods for different databases
    - a method to guess namespace of the metabolites id
```

**Comment by Unaimend** [comment]
[My current branch ](https://github.com/Porthmeus/MeMoMe/tree/dev/thomas/user-interface)  already mergred those changes

**Comment by Porthmeus** [comment]
Closed with #51
